.github/workflows/unit-tests.yml

```diff
@@ -19,8 +19,9 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
-        operating-system:
-          - "ubuntu-latest"
+
+    env:
+      SYMFONY_DEPRECATIONS_HELPER: 1
```

Comment:
there's no deprecation actually, this only happens with an old version lcobucci/jwt v3, because there's a `@param $pem` that was wrongly placed in the docblock ( the actual parameter is named $key ), symfony debug thinks it's a new argument that must be provided, and raises a deprecation warning.